Mathlib/Data/Int/Cast/Lemmas.lean

```diff
@@ -276,6 +277,15 @@ of `Multiplicative.ofAdd 1`. -/
 def zpowersHom : α ≃ (Multiplicative ℤ →* α) :=
   ofMul.trans <| (zmultiplesHom _).trans <| AddMonoidHom.toMultiplicative''
 
+/-- The equivalence `(ULift ℤ →+ G) ≃ G` for any additive group `G`. -/
+def uliftZMultiplesHom (G : Type u) [AddGroup G] : G ≃ (ULift.{u} ℤ →+ G) :=
+  (zmultiplesHom _).trans <| AddMonoidHom.precompEquiv .ulift _
+
+/-- The equivalence `(ULift (Multiplicative ℤ) →* G) ≃ G` for any group `G`. -/
+@[to_additive existing (attr := simps!)]
+def uliftZPowersHom (G : Type u) [Group G] : G ≃ (ULift.{u} (Multiplicative ℤ) →* G) :=
+  (zpowersHom _).trans <| MonoidHom.precompEquiv .ulift _
```

Comment:
I think these made more sense in the original category theory file, since the only reason you'd want them is the universe monomorphism that category theory forces.